src/IrcFeatures/DccChat.cs

```diff
@@ -130,8 +130,8 @@ internal override void InitWork(Object stateInfo)
 
             DccChatStartEvent(new DccEventArgs(this));
 
-            _sr = new StreamReader(Connection.GetStream(), Irc.Encoding);
-            _sw = new StreamWriter(Connection.GetStream(), Irc.Encoding);
+            _sr = new StreamReader(Connection.GetStream(), System.Text.Encoding.Default);
+            _sw = new StreamWriter(Connection.GetStream(), System.Text.Encoding.Default);
```

Comment:
Oh for goodness sake Im an idiot... IrcClient extends IrcCommands extends IrcConnection, check... I'll fix it :)